torch/_inductor/codegen/wrapper.py

```diff
@@ -548,6 +548,7 @@ def write_header(self) -> None:
                 import random
                 import os
                 import tempfile
+                from torch.cuda import nvtx
```

Comment:
can you add this conditionally ? see https://github.com/pytorch/pytorch/blob/dcdb25453e0ddc6a83e0052fffc544d4d03cdffd/torch/_inductor/codegen/triton.py#L2573

Reply:
@eellison done https://github.com/pytorch/pytorch/pull/130429/commits/a61c70363eb12596817828e96e96abb72e12a492